src/stratigraphy/data_extractor/data_extractor.py

```diff
@@ -63,6 +66,11 @@ def __init__(self, document: fitz.Document):
 
         self.doc = document
         self.feature_keys = read_params("matching_params.yml")[f"{self.feature_name}_keys"]
+        self.feature_fp_keys = (
+            read_params("matching_params.yml")[f"{self.feature_name}_fp_keys"]
+            if read_params("matching_params.yml")[f"{self.feature_name}_fp_keys"]
+            else []
+        )
```

Comment:
is this the same as `read_params("matching_params.yml")[f"{self.feature_name}_fp_keys"] or []` ?

Reply:
Yes, and shorter! Thanks!